Source/Scene/GlobeDepth.js

```diff
@@ -0,0 +1,235 @@
+/*global define*/
+define([
+        '../Core/Color',
+        '../Core/defined',
+        '../Core/destroyObject',
+        '../Core/PixelFormat',
+        '../Renderer/ClearCommand',
+        '../Renderer/PixelDatatype',
+        '../Shaders/PostProcessFilters/PassThrough'
+    ], function(
+        Color,
+        defined,
+        destroyObject,
+        PixelFormat,
+        ClearCommand,
+        PixelDatatype,
+        PassThrough) {
+    "use strict";
+    /*global WebGLRenderingContext*/
+
+    /**
+     * @private
+     */
+    var GlobeDepth = function(context) {
+        this._colorTexture = undefined;
+        this._depthStencilTexture = undefined;
+        this._depthStencilGlobeTest = undefined;
+
+        this.framebuffer = undefined;
+        this._copyDepthFramebuffer = undefined;
+
+        this._clearColorCommand = undefined;
+        this._copyColorCommand = undefined;
+        this._copyDepthCommand = undefined;
+
+        var supported = updateFramebuffers(this, context);
+        if (supported) {
+            updateCopyCommands(this, context);
+        }
+
+        this.supported = supported;
+    };
+
+    function destroyTextures(globeDepth) {
+        globeDepth._colorTexture = globeDepth._colorTexture && !globeDepth._colorTexture.isDestroyed() && globeDepth._colorTexture.destroy();
+        globeDepth._depthStencilTexture = globeDepth._depthStencilTexture && !globeDepth._depthStencilTexture.isDestroyed() && globeDepth._depthStencilTexture.destroy();
+        globeDepth._depthStencilGlobeTest = globeDepth._depthStencilGlobeTest && !globeDepth._depthStencilGlobeTest.isDestroyed() && globeDepth._depthStencilGlobeTest.destroy();
+    }
+
+    function destroyFramebuffers(globeDepth) {
+        globeDepth.framebuffer = globeDepth.framebuffer && !globeDepth.framebuffer.isDestroyed() && globeDepth.framebuffer.destroy();
+        globeDepth._copyDepthFramebuffer = globeDepth._copyDepthFramebuffer && !globeDepth._copyDepthFramebuffer.isDestroyed() && globeDepth._copyDepthFramebuffer.destroy();
+    }
+
+    function createTextures(globeDepth, context, width, height) {
+        globeDepth._colorTexture = context.createTexture2D({
+            width : width,
+            height : height,
+            pixelFormat : PixelFormat.RGBA,
+            pixelDatatype : PixelDatatype.UNSIGNED_BYTE
+        });
+
+        globeDepth._depthStencilTexture = context.createTexture2D({
+            width : width,
+            height : height,
+            pixelFormat : PixelFormat.DEPTH_STENCIL,
+            pixelDatatype : PixelDatatype.UNSIGNED_INT_24_8_WEBGL
+        });
+        globeDepth._depthStencilGlobeTest = context.createTexture2D({
```

Comment:
It was `depthStencilCopyTexture`. You suggested the current name in an earlier comment. How about `globeDepthTexture` or something similar?